lib/commands/context.js

```diff
@@ -277,6 +277,7 @@ helpers.setupNewChromedriver = async function setupNewChromedriver (opts, curDev
     port: opts.chromeDriverPort,
     executable: opts.chromedriverExecutable,
     adb,
+    cmdArgs: opts.chromedriverArgs,
```

Comment:
Do we need to update appium-chromedriver in package.json to accept this arg?

Reply:
@KazuCocoa appium-chromedriver already accepted it, we just had never used it before!